docs/build.md

```
@@ -55,9 +55,39 @@ Notes:
+
+The following options can be specified in the make command line using the `NAME=VAL` format.
+
+| Name       | Required | Default  | Comments |
+|------------|----------|----------|----------|
+| RPM_OUTDIR | yes      | none     | RPM repository directory to convert |
+
+The `make rpm-deb` command converts MicroShift RPMs to Debian packages. The path
+to an existing RPM repository must be specified using the mandatory `RPM_OUTDIR`
+make command line.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)